### PR TITLE
[CS-1571] Redesign QR scanning screen

### PR DESCRIPTION
**Pull request by topwebtek7**
Updated QR scanning screen

image

### CODE REVIEW

**Comment by kdowlin** [comment]
UI looks good.

File: cardstack/src/components/Sheet/Sheet.tsx [hunk 1]
```diff
@@ -10,6 +11,7 @@ export const Sheet = ({
   borderRadius = 39,
   children,
   hideHandle = false,
+  style,
```
Comment (DominicSherman) [inline cardstack/src/components/Sheet/Sheet.tsx]: We could just extend the `ContainerProps` and pass those through that we still are using the restyle library for our styling

File: src/components/walletconnect-list/WalletConnectExplainerItem.js [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
-            lineHeight="loose"
-            size="smedium"
-          >
-            {content}
-          </Text>
-        </FlexItem>
-        {children && <FlexItem marginTop={8}>{children}</FlexItem>}
-      </Column>
-    </Row>
+    <Column
```
Comment (DominicSherman) [inline src/components/walletconnect-list/WalletConnectExplainerItem.js]: Could we refactor this to use `Container` and `Text` from `@cardstack/components`?

Comment (topwebtek7) [reply src/components/walletconnect-list/WalletConnectExplainerItem.js]: @DominicSherman refactored using `Container` and `Text` from `@cardstack/components`

**Review by DominicSherman — APPROVED** [review]

**Pull request merged**